README.md

```diff
@@ -1,3 +1,28 @@
-## About
+# About
 
-This is a (very) lightweight discord clone.
+Liscord is a (very) lightweight discord clone.
+
+# Requirements
+
+* Python 3.x
+
+# Installation
+
+* git clone https://github.com/raqibahmed/liscord
+
+```bash
+pip install -r requirements.txt
+```
+```bash
+pip install phonenumbers
+```
+```bash
+.manage.py runserver
+```
+
+# Features implemented so far
+
+* Creation and Joining Servers.
+* Creation of Channels in a Servers.
+* Server Settings (Change name and delete).
+* Channel Settings (Change name and delete).
```

Comment:
User profile and adding/editing/removing messages in a channel can also be added here as a feature as those are already completed.